### ISSUE TITLE
[Snyk:  High] prototype pollution (Due: 08/28/2020)

### ISSUE THREAD

**Issue by jason-upchurch** [fecgov/fec-pattern-library#165]
### Summary
vulnerability identified by snyk  test:

```  ✗ Prototype Pollution [High Severity][https://snyk.io/vuln/SNYK-JS-LODASHMERGEWITH-174136] in lodash.mergewith@4.6.1
introduced by fec-cms@1.0.0 > sanitize-html@1.18.4 > lodash.mergewith@4.6.1
This issue was fixed in versions: 4.6.2
```

###  Steps to reproduce
`npm i && nmp run build`
`snyk test  --file=package.json`

**Completion criteria:**
- [ ] update to 4.6.2

**Comment by jason-upchurch** [comment]
@rfultz , this issue should be resolved by https://github.com/fecgov/fec-pattern-library/pull/166 as it includes `lodash.mergewith@4.6.2` in package-lock.json.

**Comment by jason-upchurch** [comment]
@lbeaufort @rfultz this issue was also resolved by #166
